G1ANT.Addon.Ocr.AbbyyFineReader/G1ANT.Addon.Ocr.AbbyyFineReader/Structures/AbbyyRowStructure.md

```diff
@@ -0,0 +1,11 @@
+# abbyyrow
+
+This structure stores information about a row on a page and has five fields:
+
+| Field   | Type    | Description                                                  |
+| ------- | ------- | ------------------------------------------------------------ |
+| `cells`  | abbyrow |             |
+| `count` | integer |             |
+| `top`   | integer |             |
+| `bottom` | integer |             |
+| `istablerow` | boolean |             |
```

Comment:
Personally I don't like the idea of delivering empty documentation.

Reply:
neither do I